packages/extension-chakra-storefront/src/components/hero/index.jsx

```diff
@@ -20,17 +20,18 @@ const Hero = ({title, img, actions, ...props}) => {
             {...props}
         >
             <Stack
-                align={'center'}
-                spacing={{base: 8, md: 10}}
+                align="center"
+                gap={{base: 8, md: 10}}
                 paddingTop={{base: 12, md: 10}}
                 paddingBottom={{base: 6, md: 10}}
                 direction={{base: 'column', lg: 'row'}}
             >
-                <Stack flex={1} spacing={{base: 5, md: 8}}>
+                <Stack flex={1} gap={{base: 5, md: 8}}>
                     <Heading
                         as="h1"
                         fontSize={{base: '4xl', md: '5xl', lg: '6xl'}}
                         maxWidth={{base: '75%', md: '50%', lg: 'md'}}
+                        lineHeight={{base: 'shorter', md: 'shorter', lg: 'shorter'}}
```

Comment:
FYI: It looks like in V2 there were default styles applied to components through the styleConfig system. But in V3 we need to be explicit

Reply:
we can do base:shorter and it will go up to lg since these values are the same, Chakra uses mobile first pattern, so it will set min-width if we set base, anything lower will use the default lineheight.